apriltags2_ros/src/common_functions.cpp

```diff
@@ -150,7 +152,7 @@ TagDetector::~TagDetector() {
   apriltag_detector_destroy(td_);
 
   // Free memory associated with the array of tag detections
-  zarray_destroy(detections_);
+  apriltag_detections_destroy(detections_);
```

Comment:
Is this really necessary? It doesn't compile for me this way.

Reply:
@Jntzko 
thank you for the comment.

Could you please show me the error log? It compiles on my computer.

Reply:
This is the output:

error: cannot convert ‘zarray_t* {aka zarray*}’ to ‘apriltag_detector_t* {aka apriltag_detector*}’ for argument ‘1’ to ‘void apriltag_detector_destroy(apriltag_detector_t*)’
   apriltag_detector_destroy(detections_);

Reply:
@Jntzko 
Did you copy the changes all add the changes manually?
If you added it manually, your code completion gave a wrong suggestion.

It is `apriltag_**detections**_destroy` instead of `apriltag_**detector**_destroy`

Reply:
Oh great, now it works, sorry ;)

Reply:
:smiley: